internal/engine/engine.go

```diff
@@ -268,27 +268,27 @@ func parseInput(input string) ([]string, map[string]string, error) {
 // executeCommand executes the parsed commands with their corresponding arguments.
 // It returns an error if the execution fails.
 func (be *BotEngine) executeCommand(
-	appID entity.PlatformID,
+	platformID entity.PlatformID,
 	callerID string,
 	commands []string,
 	args map[string]string,
 ) command.CommandResult {
 	log.Debug("execute command", "callerID", callerID, "commands", commands, "args", args)
 
 	cmd := be.getTargetCommand(commands)
-	if !cmd.HasAppID(appID) {
-		return cmd.FailedResultF("unauthorized appID: %v", appID)
+	if !cmd.HasPlatformID(platformID) {
+		return cmd.FailedResultF("unauthorized platformID: %v", platformID)
 	}
 
 	if cmd.Handler == nil {
 		return cmd.RenderHelpTemplate()
 	}
 
-	caller, err := be.GetUser(appID, callerID)
+	caller, err := be.GetUser(platformID, callerID)
 	if err != nil {
 		log.Error(err.Error())
 
-		return cmd.ErrorResult(fmt.Errorf("user is not defined in %s application", appID.String()))
+		return cmd.ErrorResult(fmt.Errorf("user is not defined in %s application", platformID.String()))
```

Comment:
```suggestion
		return cmd.ErrorResult(fmt.Errorf("user is not defined in %s platform", platformID.String()))
```
This also works (without `String`)

```
		return cmd.FailedResultF("unauthorized platformID: %v", platformID)
```